src/xcode/ENA/ENA/Resources/Localization/de.lproj/privacy-policy.html

```diff
@@ -5,7 +5,7 @@
 
 			<title>Datenschutzerklärung &bull; Corona-Warn-App</title>
 
-			<script src="/ruxitagentjs_ICA27SVfqrux_10213210506081349.js" data-dtconfig="app=ea7c4b59f27d43eb|rcdec=1209600000|featureHash=ICA27SVfqrux|vcv=2|rdnt=1|uxrgce=1|srcss=1|bp=3|srmcrv=10|cuc=cuve8t91|mel=100000|dpvc=1|ssv=4|lastModification=1620323184441|dtVersion=10213210506081349|srmcrl=1|tp=500,50,0,1|uxdcw=1500|vs=2|agentUri=/ruxitagentjs_ICA27SVfqrux_10213210506081349.js|reportUrl=/rb_3cbfc0c9-a571-451d-b533-3ebd26e9f5f5|rid=RID_1936346706|rpid=-461134329|domain=net.sap"></script><script src="/ruxitagentjs_ICA27SVfqrux_10207210127152629.js" data-dtconfig="app=ea7c4b59f27d43eb|featureHash=ICA27SVfqrux|vcv=2|rdnt=1|uxrgce=1|bp=3|cuc=cuve8t91|mel=100000|dpvc=1|lastModification=1613706421428|dtVersion=10207210127152629|tp=500,50,0,1|uxdcw=1500|vs=2|agentUri=/ruxitagentjs_ICA27SVfqrux_10207210127152629.js|reportUrl=/rb_3cbfc0c9-a571-451d-b533-3ebd26e9f5f5|rid=RID_-1896203118|rpid=259947634|domain=net.sap"></script><script src="/ruxitagentjs_ICA27SVfqrux_10205201218101503.js" data-dtconfig="app=ea7c4b59f27d43eb|featureHash=ICA27SVfqrux|vcv=2|rdnt=1|uxrgce=1|bp=3|cuc=cuve8t91|dpvc=1|lastModification=1608313592462|dtVersion=10205201218101503|tp=500,50,0,1|uxdcw=1500|vs=2|agentUri=/ruxitagentjs_ICA27SVfqrux_10205201218101503.js|reportUrl=/rb_3cbfc0c9-a571-451d-b533-3ebd26e9f5f5|rid=RID_1499435536|rpid=-978447134|domain=net.sap"></script><style>
+			<script type="text/javascript" src="/ruxitagentjs_ICA27SVfqrux_10215210506134511.js" data-dtconfig="app=ea7c4b59f27d43eb|rcdec=1209600000|featureHash=ICA27SVfqrux|vcv=2|rdnt=1|uxrgce=1|srcss=1|bp=3|srmcrv=10|cuc=cuve8t91|mel=100000|dpvc=1|ssv=4|lastModification=1620627555494|dtVersion=10215210506134511|srmcrl=1|tp=500,50,0,1|uxdcw=1500|vs=2|agentUri=/ruxitagentjs_ICA27SVfqrux_10215210506134511.js|reportUrl=/rb_3cbfc0c9-a571-451d-b533-3ebd26e9f5f5|rid=RID_917400526|rpid=-1422294221|domain=net.sap"></script><script type="text/javascript" src="/ruxitagentjs_ICA27SVfqrux_10207210127152629.js" data-dtconfig="app=ea7c4b59f27d43eb|featureHash=ICA27SVfqrux|vcv=2|rdnt=1|uxrgce=1|bp=3|cuc=cuve8t91|mel=100000|dpvc=1|lastModification=1613706421428|dtVersion=10207210127152629|tp=500,50,0,1|uxdcw=1500|vs=2|agentUri=/ruxitagentjs_ICA27SVfqrux_10207210127152629.js|reportUrl=/rb_3cbfc0c9-a571-451d-b533-3ebd26e9f5f5|rid=RID_-1896203118|rpid=259947634|domain=net.sap"></script><script type="text/javascript" src="/ruxitagentjs_ICA27SVfqrux_10205201218101503.js" data-dtconfig="app=ea7c4b59f27d43eb|featureHash=ICA27SVfqrux|vcv=2|rdnt=1|uxrgce=1|bp=3|cuc=cuve8t91|dpvc=1|lastModification=1608313592462|dtVersion=10205201218101503|tp=500,50,0,1|uxdcw=1500|vs=2|agentUri=/ruxitagentjs_ICA27SVfqrux_10205201218101503.js|reportUrl=/rb_3cbfc0c9-a571-451d-b533-3ebd26e9f5f5|rid=RID_1499435536|rpid=-978447134|domain=net.sap"></script><style>
```

Comment:
I removed the type="text/javascript" intentionally as it creates warning during html validation.